<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addGroupe</class>
 <widget class="QWidget" name="addGroupe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="idgm_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>160</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox { border: 0; }</string>
   </property>
   <property name="title">
    <string>Id Groupe :</string>
   </property>
   <widget class="QLineEdit" name="idgm">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>261</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 7px;
border: 1px solid #555555;</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="niveau_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>220</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox { border: 0; }</string>
   </property>
   <property name="title">
    <string>Niveau :</string>
   </property>
   <widget class="QComboBox" name="niveau">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 7px;
border: 1px solid #555555;</string>
    </property>
    <item>
     <property name="text">
      <string>1er</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2eme</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3eme</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QPushButton" name="cancelGRPButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>460</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 7px;
border: 1px solid #555555;</string>
   </property>
   <property name="text">
    <string>Continue</string>
   </property>
  </widget>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>30</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Candara</family>
     <pointsize>36</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{color: rgb(198, 0, 0);}</string>
   </property>
   <property name="text">
    <string>Add Groupe</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="gm">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>280</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox { border: 0; }</string>
   </property>
   <property name="title">
    <string>Groupe Module  :</string>
   </property>
   <widget class="QPushButton" name="addGRPGM">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 7px;
border: 1px solid #555555;</string>
    </property>
    <property name="text">
     <string>Include Groupe M</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="diplome_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>280</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox { border: 0; }</string>
   </property>
   <property name="title">
    <string>Diplôme :</string>
   </property>
   <widget class="QComboBox" name="diplome">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 7px;
border: 1px solid #555555;</string>
    </property>
    <item>
     <property name="text">
      <string>ING</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Master</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>LMD</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QGroupBox" name="specialite_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>340</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox { border: 0; }</string>
   </property>
   <property name="title">
    <string>Spécialité :</string>
   </property>
   <widget class="QComboBox" name="specialite">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 7px;
border: 1px solid #555555;</string>
    </property>
    <item>
     <property name="text">
      <string>INF</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>INDUS</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>MATH</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QGroupBox" name="numg_2">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>220</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox { border: 0; }</string>
   </property>
   <property name="title">
    <string>Numéro Groupe :</string>
   </property>
   <widget class="QComboBox" name="numg">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 7px;
border: 1px solid #555555;</string>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QGroupBox" name="std">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>340</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox { border: 0; }</string>
   </property>
   <property name="title">
    <string>Student  :</string>
   </property>
   <widget class="QPushButton" name="addGRPETU">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 7px;
border: 1px solid #555555;</string>
    </property>
    <property name="text">
     <string>Include Student</string>
    </property>
   </widget>
  </widget>
  <widget class="QTableView" name="tableViewUpdateGRP">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>361</width>
     <height>351</height>
    </rect>
   </property>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QPushButton" name="modifyGRPButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>460</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 7px;
border: 1px solid #555555;</string>
   </property>
   <property name="text">
    <string>Modify</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearGRPButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>460</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 7px;
border: 1px solid #555555;</string>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
